core.lua

```diff
@@ -1,4 +1,6 @@
-if WOW_PROJECT_ID ~= WOW_PROJECT_MAINLINE then return end
+local IS_RETAIL = WOW_PROJECT_ID == WOW_PROJECT_MAINLINE
+local IS_CLASSIC = WOW_PROJECT_ID == WOW_PROJECT_CLASSIC
```

Comment:
There's one more variant of Classic we need to support: `WOW_PROJECT_WRATH_CLASSIC`

Maybe for now we could just make IS_CLASSIC return true if the project is `WOW_PROJECT_WRATH_CLASSIC` OR `WOW_PROJECT_CLASSIC`?

Reply:
I added a `IS_WRATH` flag for later use in the code. So we consider all 3 editions of the game.